### PR TITLE
PR: Update Contributing guide to specify forking the Spyder repo

### PR DESCRIPTION
**Pull request by CAM-Gerlach**
## Description of Changes

A participant in our SciPy 2020 sprint commented that our contributing guide mentions cloning the Spyder repo directly, rather forking it which is necessary to create a pull request. @juanis2112 pointed this out, and as such I've updated the Contributing guide accordingly.

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: CAM-Gerlach

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -98,7 +109,7 @@ $ python runtests.py
 ```
 
 
-## Spyder Branches
+## Spyder branches
```
Comment (juanis2112) [inline CONTRIBUTING.md:112]:
```suggestion
## Spyder Branches
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -188,6 +200,9 @@ As an example, let's assume that (i) your Github user name is `myuser`; (ii) you
 where `<branch>` needs to be `1.x` if your `fix_in_spyder` branch was done against Spyder's `4.x` branch; and `master`, if you did it against our `master` branch here.
 
 
+## Submi
+
+
```
Comment (juanis2112) [inline CONTRIBUTING.md:205]:
```suggestion
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -77,7 +88,7 @@ $ python bootstrap.py --debug
 **Important Note**: To test any changes you've made to the Spyder source code, you need to restart Spyder or start a fresh instance (you can run multiple copies simultaneously by unchecking the Preferences option <kbd>Use a single instance</kbd> under <kbd>General</kbd> > <kbd>Advanced Settings</kbd> .
 
 
-##  Running Tests
+###  Running Tests
```
Comment (juanis2112) [inline CONTRIBUTING.md:91]:
```suggestion
###  Running tests
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -22,18 +23,27 @@ Most importantly, aside from the error message/traceback and the requested envir
...
+
+```bash
+$ git clone <LINK-TO-YOUR-REPO>
+```
+
+Finally, set the upstream remote to the official Spyder repo with:
 
 ```bash
-$ git clone https://github.com/spyder-ide/spyder.git
+git remote add upstream https://github.com/spyder-ide/spyder.git
```
Comment (juanis2112) [inline CONTRIBUTING.md:38]:
```suggestion
$ git remote add upstream https://github.com/spyder-ide/spyder.git
```

**Comment by CAM-Gerlach** [comment]
@ccordoba12 Please merge to `master`, I can't anymore with the branch protection changes. Thanks!

**Comment by ccordoba12** [comment]
Done! But there was a missing command that both Gurpreet and David noticed today: `git checkout 4.x` to be able to start Spyder from that branch (that's because `master` has already different dependencies from `4.x`).

@CAM-Gerlach, could you open another PR for that? Thanks!

**Comment by ccordoba12** [comment]
Sorry, I just noticed that that's already mentioned on the guide (although perhaps it should be made more explicit?)

**Pull request merged**